src/components/ContentItem/ContentItem.vue

```diff
@@ -5,8 +5,9 @@
       @click="emitClickEvent"
   >
     <div v-if="$slots.default"
-         :data-testid="`${dataTestId}-media-icon`"
          class="media"
+         :class="{ 'no-border': noBorder }"
```

Comment:
You are duplicating `class`, is this correct? Can we unify them?

Reply:
Indeed, it is correct, however it's not exactly a duplication. The first `class` (line: 8) is a classic HTML attribute for a static CSS class that should always be present, while the bound `:class` (line: 9) is dynamic and will be evaluated by the Vue runtime. Yes, it is possible to combine both into a single, complex `:class` binding, however I much prefer to avoid such complexity in favor of the more obvious/explicit split syntax. I'm happy to discuss more if you'd like, just let me know.